v2/router.go

```diff
@@ -48,9 +48,38 @@ type Procedure struct {
 	Signature string
 }
 
+// EncodingProcedure specifies a single encoding-level handler and is mapped to a TransportProcedure by the router.
+type EncodingProcedure struct {
+	// Name of the procedure.
+	Name string
+
+	// Service or empty to use the default service name.
+	Service string
+
+	// HandlerSpec specifying which handler and rpc type.
+	HandlerSpec EncodingHandlerSpec
+
+	// Encoding of the handler.
+	// (if present).
+	Encoding Encoding
+
+	// Human-readable signature of the handler.
+	Signature string
+
+	// Codec to assist mapping the encoding-level handler to transport-level handler
+	Codec InboundCodec
+}
+
+// InboundCodec helps convert the request/response bodies to & from interface{}
```

Comment:
nit: the `&` looks a little strange. Maybe,

```
// InboundCodec defines the interface YARPC uses to encode and decode request and response bodies.
```

What are your thoughts?

Reply:
sgtm